frontend-api/src/main/java/uk/gov/di/authentication/frontendapi/services/AccountInterventionsService.java

```diff
@@ -0,0 +1,129 @@
+
+    private static final String FIELD_UPDATED_AT = "updatedAt";
+    private static final String FIELD_APPLIED_AT = "appliedAt";
+    private static final String FIELD_SENT_AT = "sentAt";
+    private static final String FIELD_DESCRIPTION = "description";
+    private static final String FIELD_REPROVED_IDENTITY_AT = "reprovedIdentityAt";
+    private static final String FIELD_RESET_PASSWORD_AT = "resetPasswordAt";
+    private static final String FIELD_BLOCKED = "blocked";
+    private static final String FIELD_SUSPENDED = "suspended";
+    private static final String FIELD_REPROVE_IDENTITY = "reproveIdentity";
+    private static final String FIELD_RESET_PASSWORD = "resetPassword";
+    private static final String FIELD_INTERVENTION = "intervention";
+    private static final String FIELD_STATE = "state";
+    private static final String ERROR_MISSING_INTERVENTION =
+            "Intervention object is missing in the response";
+    private static final String ERROR_MISSING_STATE = "State object is missing in the response";
+    private static final String ERROR_FIELD_MISSING_OR_NULL = "'%s' is null or missing";
+
+    public AccountInterventionsInboundResponse sendAccountInterventionsOutboundRequest(
+            HTTPRequest request) throws UnsuccessfulAccountInterventionsResponseException {
+
+        try {
+            LOG.info("Sending account interventions outbound request");
+            var response = request.send();
+            if (!response.indicatesSuccess()) {
+                throw new UnsuccessfulAccountInterventionsResponseException(
+                        format(
+                                "Error %s when attempting to call Account Interventions outbound endpoint: %s",
+                                response.getStatusCode(), response.getContent()),
+                        response.getStatusCode());
+            }
+            LOG.info("Received successful account interventions outbound response");
+            return parseResponse(response);
+        } catch (IOException e) {
+            throw new UnsuccessfulAccountInterventionsResponseException(
+                    "Error when attempting to call Account Interventions outbound endpoint", e);
+        } catch (ParseException e) {
+            throw new UnsuccessfulAccountInterventionsResponseException(
+                    "Error parsing HTTP response", e);
+        }
+    }
+
+    private AccountInterventionsInboundResponse parseResponse(HTTPResponse response)
+            throws ParseException {
+        var contentAsString = response.getContentAsJSONObject();
+
+        var interventionJson = (JSONObject) contentAsString.get(FIELD_INTERVENTION);
+        if (interventionJson == null) {
+            LOG.debug(ERROR_MISSING_INTERVENTION);
+            throw new ParseException(ERROR_MISSING_INTERVENTION);
+        }
+        var updatedAt =
+                checkFieldNotNull(interventionJson.getAsString(FIELD_UPDATED_AT), FIELD_UPDATED_AT);
```

Comment:
Wondering here whether we can use some automatic deserialisation of json into our record object, as that would get rid of a lot of this code - also asked on slack in case people who know their way around java libraries better than me can immediately spot the way